### PR TITLE
Implement filtering by country in ;handle list

### LINKED ISSUE #248: country filter option for ;handle list

### PR DESCRIPTION
**Pull request by krofna**
Closes #248

### CODE REVIEW

File: tle/cogs/handles.py [hunk 1]
```diff
@@ -318,13 +318,13 @@ def __init__(self, bot):
         await ctx.send(msg)
 
     @handle.command(brief="Show all handles")
-    async def list(self, ctx):
+    async def list(self, ctx, *countries):
         """Shows all members of the server who have registered their handles and
         their Codeforces ratings.
```
Comment (meooow25) [inline tle/cogs/handles.py]: Describe that countries can be used to filter in help string.

File: tle/cogs/handles.py [hunk 2]
```diff
@@ -318,13 +318,13 @@ def __init__(self, bot):
...
         """
         res = cf_common.user_db.get_cf_users_for_guild(ctx.guild.id)
         users = [(ctx.guild.get_member(int(user_id)), cf_user.handle, cf_user.rating)
-                 for user_id, cf_user in res]
+                 for user_id, cf_user in res if not countries or cf_user.country in countries]
```
Comment (meooow25) [inline tle/cogs/handles.py]: Might want to accept "country" as well as "Country". In `;plot country` the countries are changed to title case before checking.

Comment (meooow25) [inline tle/cogs/handles.py]: Show error message if no users match.

File: tle/cogs/handles.py [hunk 3]
```diff
@@ -318,13 +318,13 @@ def __init__(self, bot):
...
         users = [(member, handle, rating) for member, handle, rating in users if member is not None]
         users.sort(key=lambda x: (1 if x[2] is None else -x[2], x[1]))  # Sorting by (-rating, handle)
         pages = _make_pages(users)
```
Comment (meooow25) [inline tle/cogs/handles.py]: You would want the message title to also show the countries, currently it's just "Handles of server members".

**Comment by krofna** [comment]
merged

**Pull request closed without merging**